Libraries/src/Amazon.Lambda.RuntimeSupport/Bootstrap/UserCodeInit.cs

```diff
@@ -0,0 +1,194 @@
+        {
+            if (inputExpression != null)
+            {
+                try
+                {
+                    byte[] byteArray = Encoding.ASCII.GetBytes("''");
+                    using (Stream dummyInStream = new MemoryStream(byteArray))
+                    {
+                        Expression.Lambda(inputExpression, inStreamParameter)
+                            .Compile()
+                            .DynamicInvoke(dummyInStream);
+                    }
+                }
+                catch
+                {
+                    // An exception is expected here because the dummy JSON is unlikely to match POCO
+                }
+            }
+        }
+
+        public static void InitSerializationAssembly(Expression outputExpression, ParameterExpression outStreamParameter, Type customerOutputType)
+        {
+            if (outputExpression == null || customerOutputType == null || customerOutputType == typeof(void))
+                return;
+            try
+            {
+                var customerObjectParameter = Expression.Parameter(customerOutputType, "customerObject");
+                byte[] byteArray = Encoding.ASCII.GetBytes("''");
+                using (Stream dummyInStream = new MemoryStream(byteArray))
+                {
+                    Expression.Lambda(outputExpression, customerObjectParameter, outStreamParameter)
+                        .Compile()
+                        .DynamicInvoke(null, dummyInStream);
+                }
+            }
+            catch
+            {
+                // An exception is expected here because the dummy JSON is unlikely to match POCO
+            }
+        }
+
+        public static void LoadStringCultureInfo()
+        {
+            try
+            {
+                string locale = Environment.GetEnvironmentVariable(ENVIRONMENT_VARIABLE_LANG);
+
+                if (!string.IsNullOrEmpty(locale))
+                {
+                    // The environment variable also has the text encoding (i.e. en_US.UTF-8). .NET 6
+                    // Does not accept culture names with text encoding and throw an exception. 
+                    // To avoid that strip off the text encoding.
+                    if (locale.Contains("."))
+                    {
+                        locale = locale.Substring(0, locale.IndexOf("."));
+                    }
+                    /*A dummy operation on string*/
+                    "Lambdalanguages".ToUpper(new CultureInfo(locale));
+                }
+                else
+                {
+                    /*A dummy operation on string*/
+                    "Lambdalanguages".ToUpper();
+                }
+            }
+            catch (Exception e)
+            {
+                InternalLogger.GetDefaultLogger().LogError(e, "PreJit: Error with LoadStringCultureInfo");
+            }
```

Comment:
Noting that in .NET Core 3.1 this is ignored for `CultureNotFoundException` - this would happen when `LANG` contains an invalid value for locale

Reply:
Yes, and this logging is just for RuntimeSupport logging and would let us trouble shoot if a customer noticed that the PreJitting isn't have the expected affect. Only shows up if `LAMBDA_RUNTIMESUPPORT_DEBUG` is enabled.